static/js/components/dashboard/GradeDetailPopup_test.js

```diff
@@ -5,7 +5,7 @@ import { assert } from 'chai';
 import sinon from 'sinon';
 import _ from 'lodash';
 
-import CourseSubRow from './CourseSubRow';
+import CourseSubRow from './GradeDetailPopup';
```

Comment:
yep - btw this will be implemented in a follow-up PR (I've started some of that work but it's not included here)